bindings/python/crocoddyl/multibody/actuations/floating-base.cpp

```diff
@@ -24,19 +24,31 @@ void exposeActuationFloatingBase() {
                                                    "Initialize the floating-base actuation model.\n\n"
                                                    ":param state: state of multibody system"))
       .def("calc", &ActuationModelFloatingBase::calc, bp::args("self", "data", "x", "u"),
-           "Compute the floating-base actuation signal from the control input u.\n\n"
+           "Compute the floating-base actuation signal and actuation set from the joint torque input u.\n\n"
```

Comment:
I believe `control` input fits better / is more general e.g. if the actuation is a PD controller [which we don't have yet as a model publicly but the current framework allows for this possibility]